src/store/createVault.ts

```diff
@@ -239,7 +243,18 @@ export const inputErrorsAtom = atom<VaultCreationErrors>(get => {
       selectedMetrics.totalDebt,
     ).value;
 
-    if (istAvailable < valueToReceive) {
```

Comment:
This is incorrect because `valueToReceive` is how much IST you'll get, not how much debt you'll add. After multiplying it by loan fee the resulting debt will be higher.

Reply:
Thanks for the note. The commit shouldn't be a "fix" commit because it's fixing up other code in the same PR. Consider `fixup!` and using rebase to squash before merge. Or if you want to squash this whole PR nevermind this time.

The error points to the value of having math and numerical validation be separated from the UI components and have unit tests. Particularly around boundaries and thresholds.